src/jsx/repo.jsx

```diff
@@ -292,3 +301,26 @@ var Repo = React.createClass({
     }
   }
 });
+
```

Comment:
Was surprised that React seems to have all the Array polyfills but this one.  Not terribly happy with placement of this although there wasn't a nice place to put it. I liked it better than for( i ... ). If you want it removed, I'll revise to do a plain for(...) loop instead. If you like it and have a better suggestion for location, I'll happily move.